ldb_reader.go

```diff
@@ -490,3 +570,110 @@ func (reader *LDBReader) getGetRowByKeyStmt(ctx context.Context, pk schema.Prima
 
 	return stmt, err
 }
+
+func (reader *LDBReader) watchForLDBs(ctx context.Context, dirPath string, last int64) {
+	// TODO: consider using a fs notification library instead of polling.
```

Comment:
missing a notification scares me - before we do this, figure out the guarantees

Reply:
Thinking about this it seems like the worst case scenario with polling is that you'd potentially detect > 1 new LDBs in between polling cycles. In that case you'd choose the newest one which seems ok to me, esp with a polling interval of 1s it seems unlikely to be a problem.

Reply:
The only reason we'd use a notification library is if a) we can't use a short polling interval (1s), which seems unlikely and b) we want LDBs to sync e2e in seconds

No need for either right now :)